test/test_binary_ufuncs.py

```diff
@@ -3353,7 +3353,7 @@ def test_bitwise_xor(self, device):
             ),
         )
 
-    @dtypes(torch.uint8, torch.int8, torch.int16, torch.int32, torch.int64)
+    @dtypes(torch.uint8, torch.int8, torch.int16, torch.int32, torch.int64, torch.float)
```

Comment:
can you add a todo to port this test to error_inputs in OpInfos?

Reply:
I updated this PR and removed this test

Reply:
Awesome, thanks!